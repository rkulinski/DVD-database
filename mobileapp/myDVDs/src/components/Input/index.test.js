import React from 'react';
import { shallow } from 'enzyme';
import { Input } from './';


describe('Testing Input component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <Input/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
