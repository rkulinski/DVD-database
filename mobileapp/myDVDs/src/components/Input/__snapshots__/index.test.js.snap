// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input component renders as expected 1`] = `
<Component
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "black",
        "borderWidth": 2,
        "flexDirection": "row",
        "height": 50,
        "margin": 10,
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "flex": 1,
          "fontSize": 37.5,
          "paddingLeft": 20,
        },
      ]
    }
  />
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    autoCorrect={false}
    placeholder="type undefined"
    style={
      Array [
        Object {
          "flex": 2,
          "fontSize": 37.5,
        },
      ]
    }
  />
</Component>
`;
